_layouts/eip.html

```diff
@@ -51,7 +51,7 @@ <h1 class="page-heading">
     {% elsif page.category != "ERC" %}
       EIP-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
     {% endif %}
-    <a href="{{ page["discussions-to"] | uri_escape }}" class="no-underline">
+    <a href="{{ page.discussions-to | uri_escape }}" class="no-underline">
```

Comment:
This won't be interpreted as `page.discussions` minus `to`?

Reply:
No, it will work.